#This config is for low bandwith connection like wifi and works with ~3MB/s

#Set ROS publishing rate - independet from the actual frame rate of the camera (it adjust dynamically)
frame_rate: 10.0

#Activate Downsampling and set the size of the generated voxels in meter. (0.05 = 5cm)
#This is helpful to reduce the bandwith in the network or to limit the data comming to your algorithm afterwards
downsampling: true
downsampling_voxel_size: 0.05

#Enable or Disable the projected range or/and intensity image
range_image: false
intensity_image: false